src/Psy/Readline/HoaConsole.php

```diff
@@ -94,7 +101,9 @@ public function readline($prompt = null)
      */
     public function redisplay()
     {
-        // noop
+        $current_line = $this->hoaReadline->getLine();
+        Cursor::clear('all');
```

Comment:
You should use `mb_strlen` in case of UTF-8 (assuming the terminal supports UTF-8).
What happened?